### ISSUE TITLE
Don't set values for jpg/webp quality by default

### ISSUE THREAD

**Issue by chregu** [rokka-io/rokka-dashboard#73]
New stacks shouldn't set the jpg/webp quality, rokka tries to be smart when they're not set (but of course you can set them, if you wanna have an exact quality)

**Comment by mweibel** [comment]
@chregu IIRC we use whatever https://api.rokka.io/stackoptions tells us to do. If we shouldn't set default values then that API should not have them.

**Comment by chregu** [comment]
Ah, I didn't know about that, that's not complete anyway. So if I don't set a default there, you don't set it either? But it's a general problem. The default is meant to say, the backend uses this, when you don't set it. And the backend may change it, because of better defaults. But I see now how that also could be a problem. mmmh

**Comment by mweibel** [comment]
Yeah we do the same with the stack operations (api.rokka.io/operations). We try to have everything automated as much as possible. 
I wonder if we should somehow check if the passed value is the same as default and then not send it, but that could also not match what the user wants. Or detect if the user changed the value in any way and only then submit it, otherwise leave it..
